### PR TITLE
[BREAKING] Set default workspaceName to "default" for API usage

### PR DESCRIPTION
**Pull request by d3xter666**
BREAKING CHANGE:
Set default workspaceName to "default" for API usage (https://github.com/SAP/ui5-project/pull/586)

JIRA: CPOUI5FOUNDATION-802
Relates to: https://github.com/SAP/ui5-tooling/issues/701

### CODE REVIEW

File: lib/graph/graph.js [hunk 1]
```diff
@@ -54,15 +54,12 @@ export async function graphFromPackageDependencies({
 	cwd = cwd ? path.resolve(cwd) : process.cwd();
 	rootConfigPath = utils.resolveConfigPath(cwd, rootConfigPath);
 
-	let workspace;
-	if (workspaceName || workspaceConfiguration) {
```
Comment (matz3) [inline lib/graph/graph.js:58]: `workspaceName` can still be falsy, so I think the if-clause shouldn't be removed.

We should actually also allow `null` as a value. See my code suggestion above.

File: lib/graph/graph.js [hunk 2]
```diff
@@ -29,8 +29,8 @@ const log = getLogger("generateProjectGraph");
  * @param {string} [options.versionOverride] Framework version to use instead of the one defined in the root project
  * @param {string} [options.resolveFrameworkDependencies=true]
  * 		Whether framework dependencies should be added to the graph
- * @param {string} [options.workspaceName]
- * 		Name of the workspace configuration that should be used if any is provided
+ * @param {string} [options.workspaceName=default]
```
Comment (matz3) [inline lib/graph/graph.js:32]: We should also make clear that with `null` the usage of the default workspace can be skipped. The CLI currently already passes `null` if `--no-workspace` option is set.

```suggestion
 * @param {string|null} [options.workspaceName=default]
```

File: lib/graph/graph.js (hunk 1 above)
Comment (d3xter666) [reply lib/graph/graph.js]: Thanks! I have updated the tests to reflect that

File: lib/graph/graph.js [hunk 3]
```diff
@@ -43,7 +43,7 @@ const log = getLogger("generateProjectGraph");
 export async function graphFromPackageDependencies({
 	cwd, rootConfiguration, rootConfigPath,
 	versionOverride, cacheMode, resolveFrameworkDependencies = true,
-	workspaceName /* TODO 4.0: default workspaceName to "default" ? */,
+	workspaceName="default",
```
Comment (matz3) [inline lib/graph/graph.js:46]: There's no test assertion that makes sure "default" is used as default value. Renaming this string does not fail any test case in `test/lib/graph/graph.js`.

**Review by matz3 — CHANGES_REQUESTED** [review]

File: lib/graph/graph.js (hunk 3 above)
Comment (d3xter666) [reply lib/graph/graph.js:46]: Thanks! I have created a test for the "default" workspace name

**Review by RandomByte — APPROVED** [review]
LGTM

**Pull request merged**